addons/mail/views/mail_activity_views.xml

```diff
@@ -162,6 +162,30 @@
         </field>
     </record>
 
+    <record id="mail_activity_view_form" model="ir.ui.view">
+        <field name="name">mail.activity.view.form</field>
+        <field name="model">mail.activity</field>
+        <field name="priority">21</field>
+        <field name="mode">primary</field>
+        <field name="inherit_id" ref="mail.mail_activity_view_form_popup"/>
+        <field name="arch" type="xml">
+            <sheet position="before">
+                <header>
+                    <button attrs="{'invisible': [('force_next', '=', True)]}" string="Mark as Done" name="action_done"
+                        type="object" class="btn-secondary"
+                        context="{'mail_activity_quick_update': True}"/>
```

Comment:
Nothing happen when using the mark as done button. (the current activity stays visible even though it was deleted, which is confusing).